courses/free-courses-en.md

```diff
@@ -947,6 +947,7 @@
 
 ### Spark
 
+* [Learn Spark](https://www.udacity.com/course/learn-spark-at-udacity--ud2002) - INSIGHT, `tch.:` David Drummond, `tch.:` Judit Lantos (Udacity)
```

Comment:
```suggestion
* [Learn Spark](https://www.udacity.com/course/learn-spark-at-udacity--ud2002) -  David Drummond, Judit Lantos (Udacity)
```
sorry for the back and forth - we're still figururing out the best way to denote contributor roles. In this case "Insight" is a sponsor or publisher more than a contributor, which we usually don't list.

Reply:
Sure, committed the change.